src/components/fields/image-upload/image-review/image-thumbnails/image-thumbnails.styles.ts

```diff
@@ -55,7 +55,7 @@ export const LoadingDot = styled.div`
 	animation: ${dotMoveKeyframe} 1.35s infinite linear;
 	opacity: 0.25;
 	background: ${Colour["bg-inverse-subtle"]};
-	margin: 0.125rem;
+	${Spacing["spacing-4"]};
```

Comment:
missing `margin` property, and should stick to 2px as the animation looks bad now

can revert this line for now since DS doesn't have a 2px